### DISCUSSION TITLE
Phone 2FA is Required by Oct 24 2024, but I don't have a phone.

### DISCUSSION THREAD

**Discussion by P1xel10** [community/community#136805] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Today I got emailed that I need to activate 2FA. I was going to try it at first but I realized that you needed a phone for 2FA. I do not have a phone. How else can I continue using Github and activate 2FA another way? I feel like many other people have this issue.

**Comment by Pxl12** [comment]
This isn't an answer but I have the same issue. I was trying to look for an answer for this question too but I stumbled upon this. If anybody can respond please do so.

**Reply by Pxl12** [reply]
Ill link it on my profile too not like anybodys following me though, hopefully someone looks at this.

**Reply by ghost** [reply]
Please help.Attak my phone.Haker!Change softwear bad

**Comment by davevad93** [comment, marked as answer]
Hi @P1xel10 ,

By using the Authenticator App option you should be able to use 2FA with a TOTP browser extension that generates the codes in your browser without using your phone.

![00000](https://github.com/user-attachments/assets/889dd25c-0548-4c6e-bf77-fc7bdd9470b8)

**Reply by P1xel10** [reply]
Thank you so much!

**Reply by EliasZaman** [reply]
Thanks for two factor authenticator

**Comment by Jhons363** [comment]
#136805

**Comment by ALNAHER1** [comment]
> #136805

@Jhons363

**Comment by ALNAHER1** [comment]
> #136805

**Comment by Pimutt98** [comment]
Brak dostępu do numeru telefonu
